### PR TITLE
Favourites

### PR DESCRIPTION
**Pull request by gillianroberts1**
- add to basket and remove from basket connected to backend and working.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -41,6 +113,16 @@ function App() {
...
+  // const removeFromFavourites = (removed) => {
+  //   const updatedFaves = [];
+  //   for (let recipe of favouriteRecipes) {
+  //     if (recipe.meal.name !== removed.name) {
+  //       updatedFaves.push(recipe);
+  //     }
+  //   }
+  //   setFavouriteRecipes(updatedFaves);
+  // };
+
```
Comment (ajrussellaudio) [inline client/src/App.js:125]: Ugh, commented code

**Review by ajrussellaudio — CHANGES_REQUESTED** [review]
Fix this

File: client/src/App.js (hunk 1 above)
Comment (ajrussellaudio) [reply client/src/App.js:125]: This was me writing quickly for an example, sorry to sound like a douche :)

**Review by samjaygill — APPROVED** [review]

**Review by R-McCluskey — APPROVED** [review]

**Comment by R-McCluskey** [comment]
Good job squad

**Pull request merged**